### ISSUE TITLE
`if...` followed immediately by block without `else`

### ISSUE THREAD

**Issue by cramertj** [rust-lang/rust-clippy#3044]
I don't know how to make a good lint for this without annoying false-positives, but I just had a bug that looked like this:

```rust
if ... {
  // do something
} {
  // do something else
}
```
It took me a while to realize that I forgot the word `else`.

**Comment by oli-obk** [comment]
We already have a lint for a missing comma in

```rust
[
    1, 2, 3,
    -2, 5, 3
    -1, 2, 9,
]
```

This is basically the same situation. If no macros are involved, and there's an `if ... { ... }` followed by a `{` without a newline, complain about it.
